mlfinlab/tests/test_labels.py

```diff
@@ -11,11 +11,12 @@
 from mlfinlab.filters.filters import cusum_filter
 from mlfinlab.labeling.labeling import add_vertical_barrier, get_events, get_bins, drop_labels
 from mlfinlab.util.volatility import get_daily_vol
+from mlfinlab.labeling import trend_scanning_labels
 
 
 class TestChapter3(unittest.TestCase):
```

Comment:
The trend scanning labels should have their own class. They are not part of Chapter 3.